internal/models/rbac_policy.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2021 The AtomCI Group Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package models
+
+import (
+	"time"
+)
+
+type RbacPolicy struct {
+	Addons
+	PType string `orm:"column(p_type);" json:"p_type"`
+	V0    string `orm:"column(v0)" json:"v0"`
+	V1    string `orm:"column(v1)" json:"v1"`
+	V2    string `orm:"column(v2)" json:"v2"`
+	V3    string `orm:"column(v3)" json:"v3"`
+	V4    string `orm:"column(v4)" json:"v4"`
+	V5    string `orm:"column(v5)" json:"v5"`
+}
+
+func (t *RbacPolicy) TableName() string {
+	return "rbac_policy"
+}
+
+func Policy() []RbacPolicy {
```

Comment:
when adding system role, it will be `GenerateCasbinrules` (based on route items generate RBAC  policy and save it into rbac_policy tables), so `Policy()` can be removed?